examples/finance/bs_ivbp.ipynb

```diff
@@ -37,13 +37,12 @@
    ],
```

Comment:
<p>Should this notebook be renamed "01_black_scholes" for consistency with other tutorials?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>